src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThrough5.java

```diff
@@ -0,0 +1,67 @@
+/*
+FallThrough
+checkLastCaseGroup = (default)false
+reliefPattern = (default)falls?[ -]?thr(u|ough)
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.fallthrough;
+
+public class InputFallThrough5 {
+
+    public Object visit(Object node, Void data) {
+        Object operator = null;
+
+
+        Object operand = node.toString();
+        Object operandValue = operand.toString();
+        if (operandValue == null) {
+            return null;
+        }
+
+        switch (operator.toString()) {
+        case "122":
+            return unaryPromotion(operandValue);
+        case "99": {
+            Number promoted = unaryPromotion(operandValue);
+            if (promoted == null) {
+                return null; // single line comment test for slist
+            } else if (promoted instanceof Integer) {
+                return -promoted.intValue();
+            } else if (promoted instanceof Long) {
+                return -promoted.longValue();
+            } else if (promoted instanceof Float) {
+                return -promoted.floatValue();
+            } else {
+                assert promoted instanceof Double;
+                return -promoted.doubleValue();
+            }
```

Comment:
we do not need much `else if` one or two is enough.

Reply:
done